### PR TITLE
0.2/691 expected input behaviours

### PR DESCRIPTION
**Pull request by alexalala**
https://trello.com/c/CDu9cD7D/691-expected-input-behaviours-getting-keyboards-out-of-the-way

🍫

### CODE REVIEW

File: resources/views/service/create_registration.blade.php [hunk 1]
```diff
@@ -197,6 +197,16 @@ function() {
                 });
             }
         );
+
+        // If enter is pressed, keyboard is hidden on iPad and form submit is disabled
+        $(document).on('keyup keypress', function(e) {
```
Comment (charlesstrange2) [inline resources/views/service/create_registration.blade.php]: Is it useful to have enter key in the "other collectors" hit the "+" button, or taborder next?

File: resources/views/service/edit_registration.blade.php [hunk 2]
```diff
@@ -236,5 +234,15 @@ function () {
             }
         );
 
+        // If enter is pressed, keyboard is hidden on iPad and form submit is disabled
+        $(document).on('keyup keypress', function(e) {
```
Comment (charlesstrange2) [inline resources/views/service/edit_registration.blade.php]: Likewise, is it worth having it add people or tabnext?

**Review by charlesstrange2 — COMMENTED** [review]
see comments :-/

File: resources/views/service/create_registration.blade.php (hunk 1 above)
Comment (alexalala) [reply resources/views/service/create_registration.blade.php]: yep! Will amend so that it only happens in the inputs with no direct submit

**Review by charlesstrange2 — APPROVED** [review]
Looks :cool:
:shipit:

**Pull request merged**